### ISSUE TITLE
To Check `Setter Basket Price`

### ISSUE THREAD

**Issue by balqaasem** [setheum/setheum#305]
```
let peg_id_one = T::GetPegIdOne;
for peg_id_one in T::SetterBasketPegs::get() {
  let peg_one_price = T::Prices::get_fiat_price(peg_id_one);
peg_one_price
}

let peg_id_two = T::GetPegIdTwo;
for peg_id_two in T::SetterBasketPegs::get() {
  let peg_two_price = T::Prices::get_fiat_price(peg_id_two);
peg_two_price
}

let total_basket_worth: Price = peg_one_price + peg_two_price;
let currencies_amount: Balance = 2;

Self::aggregate_setter_price(&total_basket_worth, &currencies_amount);;

**Comment by balqaasem** [comment]
Closes in #307

**Comment by balqaasem** [comment]
Closes in #307
